src/screens/ContactScreen/index.js

```diff
@@ -212,11 +228,13 @@ const ContactScreen = ({navigation}) => {
         // onPress={handleSearch}
       />
 
-      <View>
-        {selectedUsers && (
-          <ContactPreview users={selectedUsers} onPress={(user) => handleSelected(user)} />
-        )}
-      </View>
+      {sourceScreen !== ANONYMOUS && (
+        <View>
+          {selectedUsers && (
+            <ContactPreview users={selectedUsers} onPress={(user) => handleSelected(user)} />
+          )}
+        </View>
+      )}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The display of the `ContactPreview` component now depends on whether the source screen is `ANONYMOUS` or not. If it's not, the component is displayed; otherwise, it's not. This seems to be a reasonable change if anonymous users are not supposed to see contact previews.
```diff
+ {sourceScreen !== ANONYMOUS && (
+   <View>
+     {selectedUsers && (
+       <ContactPreview users={selectedUsers} onPress={(user) => handleSelected(user)} />
+     )}
+   </View>
+ )}
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->